### PR TITLE
Portal interaction and scroll fixes

### PR DESCRIPTION
**Pull request by sandratoh**
- Fix toast overlay from blocking other elements
- Add scroll behaviour to Create New + Change App Type modals
- Changes to parcel owner dropdown if owner is added already
- Collapse panel if no other parcels in application

### CODE REVIEW

File: portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.ts [hunk 1]
```diff
@@ -79,11 +79,35 @@ export class CreateApplicationDialogComponent implements OnInit, AfterViewChecke
 
   onReadMoreClicked() {
     this.readMoreClicked = !this.readMoreClicked;
```
Comment (dhaselhan) [inline portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.ts:81]: These 4 all look the same to me, can you DRY it up?

File: portal-frontend/src/app/features/edit-submission/other-parcels/other-parcels.component.ts [hunk 2]
```diff
@@ -254,7 +254,6 @@ export class OtherParcelsComponent extends StepComponent implements OnInit, OnDe
           } else {
             this.hasOtherParcelsInCommunity.patchValue('true');
             this.formDisabled = false;
-            await this.setHasOtherParcelsInCommunity(true);
```
Comment (dhaselhan) [inline portal-frontend/src/app/features/edit-submission/other-parcels/other-parcels.component.ts:257]: Can you explain why we don't want to do this?

Comment (sandratoh) [reply portal-frontend/src/app/features/edit-submission/other-parcels/other-parcels.component.ts:257]: This was resetting and clearing the form when user wanted to cancel removing parcel.

File: portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.ts (hunk 1 above)
Comment (sandratoh) [reply portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.ts:82]: Updated!

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**